js/app.js

```diff
@@ -132,6 +132,19 @@ var TwineApp = Backbone.Marionette.Application.extend(
   },
 
   /**
+   * Exports a JSON configuration for DoSomething.org SMS games.
+   *
+   * @param story Story model to export
+   */
+  exportSmsGame: function (story) {
+    var name = story.get('name');
+    var data = story.publish(null, null);
+    
+    var formatted = exportsms.format(data);

```

Comment:
Thought about it. I didn't because I was following the all lowercase precedent set by other js libs like ui.js, apppref.js, etc. But then went camelcase on the method name here because that's how all the other methods in app.js were done, like `publishStory`.